src/commands/kick/kick.ts

```diff
@@ -0,0 +1,61 @@
+import { Command, SafetyJim } from '../../safetyjim/safetyjim';
+import * as Discord from 'discord.js';
+
+class Kick implements Command {
+    public usage = 'kick - boot the naughty people <:';
+
+    // tslint:disable-next-line:no-empty
+    constructor(bot: SafetyJim) {}
+
+    public run(bot: SafetyJim, msg: Discord.Message, args: string): boolean {
+        this.runAsync(bot, msg, args);
+        return;
+    }
+
+    private async runAsync(bot: SafetyJim, msg: Discord.Message, args: string): Promise<void> {
+        args = args.split(' ').slice(1).join(' ');
+
+        if (msg.mentions.users.size === 0) {
+            msg.channel.send('You need to mention the user to kick.');
+            return;
+        }
+
+        let member = msg.guild.member(msg.mentions.users.first());
+
+        if (!member || !member.kickable || msg.member.highestRole.comparePositionTo(member.highestRole) <= 0) {
+            msg.channel.send('The specified member is not kickable.');
+            return;
+        }
+
+        // tslint:disable-next-line:max-line-length
+        member.kick(args || 'No reason specified'); // Audit log compatibility :) (Known Caveat: sometimes reason won't appear, or add if reason has symbols.)

```

Comment:
can we log the kick in console as well? something like `bot.log.info(\`Kicked user "${member.tag}" in guild "${msg.guild.name}"');`